### DISCUSSION TITLE
I'm making a submod, and wanted to know how to create an input for a name and like that

### DISCUSSION THREAD

**Discussion by Eddiehere** [Monika-After-Story/MonikaModDev#9786] [QA - code and content creation (technical)]
I'm not the best at explaining, but I'll try, I'm making a submod that is about when Monika 'crosses over'. E she asks if the player wants a pet. That's it, and she asks how the player wants to name it. So I wanted to know how to put an input or smth. How do I do it?

**Comment by Booplicate** [comment]
Check [this](https://www.renpy.org/dev-doc/html/input.html#renpy.input).

**Reply by Eddiehere** [reply]
Thank you! I'm kind of new to this, so it helps a lot!
